### ISSUE TITLE
Skyward Sword always corrupted data

### ISSUE THREAD

**Issue by CrappingRainbows** [J-D-K/JKSV#192]
So I changed Switch's and when I moved to the new one (tried two separate Switch's), and no matter what I do, the game says the save data is corrupted. 

Things I've tried:
- Installing different base files.
- Using different save files (including other people's back-ups).
- Reinstalling game.
- Restoring file before and after updating game.
- Using different consoles. 
- Updating JKSV.

Please help, I have 30+ hours sitting there waiting for me to come back.

**Comment by impeeza** [comment]
you need to be on the exactly same Update level for the game.

if you create the backup on a version 1.0.0 you must to restore the save backup on the same update level. or if you make the backup on version 1.0.1 the new have to be the same.

**Comment by CrappingRainbows** [comment]
Yeah I understand that and I've tried restoring it to both the 1.0.0 and the 1.0.1, neither work. As far as I know there's only two versions of the game and I've tried both.

**Comment by CrappingRainbows** [comment]
I do need support, I thought that’s what this was lol.

I have tried other people’s saves from online who other people have verified it works, but I cannot get it going for whatever reason.

I’ll try gbatemp

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by impeeza** [comment]
Exactly. You are asking for support but this space is for bug report. For support is better discord or GBATemp they are made for that. On GBATemp there's a thread of JKSV and you can port your saves for more people helping

**Comment by J-D-K** [comment]
I was playing SS when making a lot of the recentish changes and know it was working then. I might have to do some investigating with TotK when I get a chance to too. Hearing there might be problems there too? Might consider dropping the multi-threaded file read/write, since that's the biggest difference that might be causing issues.
